.github/workflows/beta.yml

```diff
@@ -4,27 +4,50 @@ on:
   push:
     branches:
       - 'beta'
+  workflow_dispatch: 
 
+env:
+  IMAGE: todrob/searcharr
 jobs:
   docker:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
why bother specifying a version?

Reply:
i saw using latest sometimes gives ubuntu 22.04 so i used 22.04 image flag in github actions.